features/raadpleeg-bewoning-met-periode/bewoning-vaste-volgorde-mogelijke-bewoner.feature

```diff
@@ -0,0 +1,169 @@
+# language: nl
+@api
+Functionaliteit: De sortering van mogelijke bewoners op vaste volgorde
+Een mogelijke bewoner is een persoon die (voor een deel) van de gevraagde periode in de onzekerheidsperiode van de gevraagde adreshouding valt
+
+  Achtergrond:
+    Gegeven adres 'A1' heeft de volgende gegevens
+      | gemeentecode (92.10) | identificatiecode verblijfplaats (11.80) |
+      |                 0800 |                         0800010000000001 |
+    En adres 'A2' heeft de volgende gegevens
+      | gemeentecode (92.10) | identificatiecode verblijfplaats (11.80) |
+      |                 0800 |                         0800010000000002 |
+
+  Regel: Binnen een bewoning samenstelling worden de mogelijke bewoners in een vaste volgorde gesorteerd.
```

Comment:
Dat mogelijke bewoners op een vast volgorde wordt gesorteerd staat al in Functionaliteit. Is het nodig om dat hier weer te herhalen?
Verder is deze regel naar mijn mening een samenvoeging van vier regels. Als deze regel als vier regels wordt beschouwd met een bijbehorende scenario om het te illustreren, dan is het volgens mij niet nodig om bij de scenario weer uitleg toe te moeten voegen